roles/artifacts/tasks/environment.yml

```diff
@@ -24,3 +24,29 @@
       ip ro ls > {{ cifmw_artifacts_basedir }}/artifacts/ip-network.txt
       ip rule ls >> {{ cifmw_artifacts_basedir }}/artifacts/ip-network.txt
       ip -j -p link ls >> {{ cifmw_artifacts_basedir }}/artifacts/ip-network.txt
+
+- name: Find yum repos files
```

Comment:
Wondering why you want to find and loop on the files instead of copying the directory directly? Care to explain a bit? Why won't a simple `cp -r /etc/yum.repos.d {{ cifmw_architect_basedir }}/artifacts/yum_repos` in the above `ansible.builtin.shell` be sufficient?

Reply:
Sorry, I thought too much on this task not sure why. Yes, simple cp can do the job. Thank you @pablintino @cjeanner :-)